templates/components/draft_results.html

```diff
@@ -0,0 +1,32 @@
+{% macro draftResults(selections) -%}
+
+
+    <link rel= "stylesheet" type= "text/css" href= "{{ url_for('static',filename='styles/draft_results.css') }}">
+
+
+    <!-- The full draft board. can we make this a max height and then have a scrollbar? -->
+    Picks
+    <table class='table'>
```

Comment:
recommend avoiding using html tags for class names. easy to have a typo like "table" vs ".table" in the css file